quickstart.rst

```diff
@@ -234,12 +234,14 @@ Interacting with a Flux Session
 
 There are several low-level commands of interest to interact with a Flux
 instance. For example, to view the total resources available in the current
-instance, ``flux hwloc info`` may be used:
+instance, ``flux resource status`` may be used:
```

Comment:
`flux resource status` doesn't give you the total resources available in the instance (just the status of the execution targets). Maybe change to "to view the status of resources in the current instance", or replace with `flux resource list` and drop the "To view the scheduling state of resources".

This might actually be a good place to introduce `flux uptime` or `flux top` also.

Reply:
Hm, also, these aren't exactly low-level commands anymore. I wonder if instead of worrying about this now, we should merge this PR as is and come back around to "refresh" this entire document.